plugins/aladino/functions/hasMissingReviews.go

```diff
@@ -0,0 +1,93 @@
+// found in the LICENSE file.
+
+package plugins_aladino_functions
+
+import (
+	"log"
+
+	"github.com/reviewpad/reviewpad/v3/lang/aladino"
+	"github.com/reviewpad/reviewpad/v3/utils"
+)
+
+func HasMissingReviews() *aladino.BuiltInFunction {
+	return &aladino.BuiltInFunction{
+		Type: aladino.BuildFunctionType([]aladino.Type{}, aladino.BuildBoolType()),
+		Code: hasMissingReviewsCode,
+	}
+}
+
+func hasMissingReviewsCode(e aladino.Env, _ []aladino.Value) (aladino.Value, error) {
+	log.Printf("Running builtin hasMissingReviewsCode")
+
+	prNum := utils.GetPullRequestNumber(e.GetPullRequest())
+	owner := utils.GetPullRequestBaseOwnerName(e.GetPullRequest())
+	repo := utils.GetPullRequestBaseRepoName(e.GetPullRequest())
+
+	requestedUsers := e.GetPullRequest().RequestedReviewers
+	requestedTeams := e.GetPullRequest().RequestedTeams
+
+	// Whenever a review is request to a team the name of the team will appear on "Teams" field.
+	// However, once a review is done by a user on that team, the name of the user will appear on "Users" field and the team name will disappear from "Teams" field.
+	// So, whenever a team name appears on "Teams" field, it means that the review is not done yet.
+	if len(requestedTeams) > 0 {
+		return aladino.BuildBoolValue(true), nil
+	}
+
+	// Whenever a review is request to a user and the reviews wasn't yet performed, the name of the user will appear on "Users" field.
+	// So, whenever a user name appears on "Users" field, it means that the review is not done yet.
+	if len(requestedUsers) > 0 {
+		return aladino.BuildBoolValue(true), nil
+	}
+
+	commits, err := utils.GetPullRequestCommits(e.GetCtx(), e.GetClient(), owner, repo, prNum)
+	if err != nil {
+		return nil, err
+	}
+
+	// When there are no commits, there's nothing to review.
+	if len(commits) == 0 {
+		log.Printf("[WARN] No commits found for PR %s/%s#%d.", owner, repo, prNum)
+		return aladino.BuildBoolValue(false), nil
+	}
+
+	// We will check the last commit regardless of being the author.
+	// However, it could make sense to check only the author last commit.
+	latestCommitDate := commits[len(commits)-1].Commit.Committer.Date
+
+	reviews, err := utils.GetPullRequestReviews(e.GetCtx(), e.GetClient(), owner, repo, prNum)
+	if err != nil {
+		return nil, err
+	}
+
+	reviewValidityByUser := make(map[string]bool)
+
+	// Foreach review extract the validity of the review.
+	// A review is only valid if performed after the last commit.
+	// This covers the case when the author has performed the requested changes but didn't explicitly request for a new review.
+	for _, review := range reviews {
+		userLogin := review.User.GetLogin()
+
+		// Skip author's reviews
+		if userLogin == e.GetPullRequest().GetUser().GetLogin() {
+			continue
+		}
+
+		// Skip users with a valid review
+		if reviewValidityByUser[userLogin] {
+			continue
+		}
+
+		reviewValidityByUser[review.User.GetLogin()] = review.SubmittedAt.After(*latestCommitDate)
```

Comment:
```suggestion
		reviewValidityByUser[userLogin] = true
```

It's unclear to me if we shouldn't be checking if the status of the review is approved or requested changes. The built-in is suppose to check if there is no review at all?